_data/companies.yml

```diff
@@ -2734,3 +2734,10 @@
   visitors: Restricted
   events: Restricted
   last_update: 2020-03-13
+ 
+ - name: Quicken Loans
+  wfh: Encouraged
+  travel: Restricted
+  visitors: "?"
```

Comment:
@waterskier2007, `vistors` should be listed as `Restricted` per the company email we received yesterday.

Reply:
updated

Reply:
🚀